.drone.jsonnet

```diff
@@ -252,4 +252,28 @@ local mac_builder(name,
   // Macos builds:
   mac_builder('macOS (Release)'),
   mac_builder('macOS (Debug)', build_type='Debug'),
+
+  // iOS static lib build
+  {
+    kind: 'pipeline',
+    type: 'exec',
+    name: 'iOS static lib',
+    platform: { os: 'darwin', arch: 'amd64' },
+    steps: [
+      { name: 'submodules', commands: submodule_commands },
+      {
+        name: 'build',
+        environment: { SSH_KEY: { from_secret: 'SSH_KEY' } },
+        commands: [
+          'echo "Building on ${DRONE_STAGE_MACHINE}"',
+          // If you don't do this then the C compiler doesn't have an include path containing
+          // basic system headers.  WTF apple:
+          'export SDKROOT="$(xcrun --sdk macosx --show-sdk-path)"',
```

Comment:
we should also put a precondition in `ios.sh` that makes it die if `$SDKROOT` isnt set.

Reply:
It's up to you to set up your compiler environment properly before compiling things (as is done here for drone). SDKROOT is one such way to do it for stock xcode clang, but if you want to use some other build mechanism then so be it.